fy22_cecil_b_moore/.gitignore

```diff
@@ -0,0 +1,2 @@
+database.ini
```

Comment:
nested `.gitignore` files aren't necessary. Use the one at the root level in the repo.